website/docs/r/policy_ipsec_vpn_service.html.markdown

```diff
@@ -37,7 +37,8 @@ The following arguments are supported:
 * `description` - (Optional) Description of the resource.
 * `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
 * `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
-* `locale_service_path` - (Required) Path of the gateway locale service associated with the IPSec VPN Service.
+* `locale_service_path` - (Optional) Path of the gateway locale service associated with the IPSec VPN Service.
```

Comment:
We should mark this as Deprecated (same for other resources)

Reply:
done